### PR TITLE
docs: modified the local engine name for folders, topics to local storage

### PR DESCRIPTION
**Pull request by datacore-gthomas**

### CODE REVIEW

**Comment by datacore-gthomas** [comment]
Hello @avishnu, please review the changes made to the following sections and share your feedback/approve the PR if it's good to go. Thanks!

@balaharish7 @Abhinandan-Purkait 

Note: Bala is working on the replicated engine section.

File: docs/main/concepts/architecture.md
Comment (avishnu) [inline docs/main/concepts/architecture.md:68]: Change 'top of the local storage' to 'top of the locally attached storage'

Comment (avishnu) [reply docs/main/concepts/architecture.md:68]: 'light-weight storage controller software'

File: docs/main/concepts/architecture.md
Comment (avishnu) [inline docs/main/concepts/architecture.md:159]: 'Local or Replicated Storage' as in L in upper-case

File: docs/main/concepts/data-engines/data-engines.md
Comment (avishnu) [inline docs/main/concepts/data-engines/data-engines.md:129]: Remove 'are those that'

File: docs/main/concepts/data-engines/local-storage.md
Comment (avishnu) [inline docs/main/concepts/data-engines/local-storage.md:12]: Remove 'or Local Volume'

Comment (avishnu) [reply docs/main/concepts/data-engines/local-storage.md:12]: 'a locally mounted storage device'

File: docs/main/concepts/data-engines/local-storage.md
Comment (avishnu) [inline docs/main/concepts/data-engines/local-storage.md:90]: - OpenEBS Local Storage is not highly available and cannot sustain node failure.
- OpenEBS Local PV Hostpath does not support snapshots and clones.

File: docs/main/introduction-to-openebs/features.mdx
Comment (avishnu) [inline docs/main/introduction-to-openebs/features.mdx:57]: Let's remove the 'and there is no limit on the number of snapshots'

Comment (avishnu) [reply docs/main/introduction-to-openebs/features.mdx:57]: backups instead of back-ups
read-only instead of read only

File: docs/main/introduction-to-openebs/introduction-to-openebs.md
Comment (avishnu) [inline docs/main/introduction-to-openebs/introduction-to-openebs.md:39]: when backed by LVM or ZFS stack

File: docs/main/introduction-to-openebs/introduction-to-openebs.md
Comment (avishnu) [inline docs/main/introduction-to-openebs/introduction-to-openebs.md:43]: This sentence is no longer applicable, can be removed

File: docs/main/introduction-to-openebs/introduction-to-openebs.md
Comment (avishnu) [inline docs/main/introduction-to-openebs/introduction-to-openebs.md:41]: Remove Distributed from all places. Distributed and Replicated are not the same.

File: docs/main/introduction-to-openebs/introduction-to-openebs.md
Comment (avishnu) [inline docs/main/introduction-to-openebs/introduction-to-openebs.md:44]: write the data to the NVMe-TCP target that...

File: docs/main/introduction-to-openebs/introduction-to-openebs.md
Comment (avishnu) [inline docs/main/introduction-to-openebs/introduction-to-openebs.md:45]: Let's remove this line.

File: docs/main/user-guides/local-storage-user-guide/additional-information/kb.md
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/additional-information/kb.md:212]: Let's remove the stuff related to openshift for the time-being. We shall add later once we test with the new version.

File: docs/main/concepts/data-engines/data-engines.md
Comment (datacore-gthomas) [reply docs/main/concepts/data-engines/data-engines.md:132]: Yes, Bala is working on updating the replicated engine term with replicated storage. The changes should reflect once his PR is merged.

File: docs/main/introduction-to-openebs/introduction-to-openebs.md
Comment (avishnu) [inline docs/main/introduction-to-openebs/introduction-to-openebs.md:51]: Revert all the above 3 lines

File: docs/main/introduction-to-openebs/introduction-to-openebs.md
Comment (avishnu) [inline docs/main/introduction-to-openebs/introduction-to-openebs.md:8]: Leave in this line as Replicated

Comment (datacore-gthomas) [reply docs/main/introduction-to-openebs/introduction-to-openebs.md]: done.

**Review by avishnu — APPROVED** [review]
lgtm

**Pull request merged**
